src/iframe/browser/tabs/tabs.mock.ts

```diff
@@ -1,18 +1,23 @@
 import { getEmbeddedAncestorOrigin } from "~utils/embedded/embedded.utils";
+import { isExternalURL } from "~utils/urls/isExternalURL";
 
 export const tabs = {
   create: async ({ url }) => {
     if (process.env.NODE_ENV === "development")
       console.log(`tabs.create({ ${url} })`);
 
+    const externalUrl = isExternalURL(url);
+
     // URL =
     // browser.runtime.getURL("tabs/welcome.html")
     // browser.runtime.getURL("tabs/dashboard.html#/contacts")
     // browser.runtime.getURL("assets/animation/arweave.png");
     // browser.runtime.getURL("tabs/auth.html")}?${objectToUrlParams(...)}
     // `tabs/dashboard.html#/apps/${activeApp.url}`
 
-    if (url === "tabs/welcome.html") {
+    if (externalUrl) {
+      window.open(url, "_blank");
```

Comment:
@pawanpaudel93 Where is this used? Was there somewhere where we were not opening a page, but we should?

Reply:
@Danziger External links in explore page, Viewblock, ao.link links etc.